CoWorkingSpace/src/Customer.java

```diff
@@ -0,0 +1,68 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Customer{
+    private Admin admin;
+    private List<WorkSpace> work;
+
+    public Customer(Admin admin){
+        this.admin = admin;
+        this.work = new ArrayList<>();
```

Comment:
It would be better without using Admin in Customer class, ex. you can create extra class which will be responsible for storing data (in your case the list of workSpaces and list of reservations).